cafe_information

```diff
@@ -0,0 +1,21 @@
+create table CafeInSanFrancisco (
+	name varchar(32) NOT NULL,
+	address varchar(64) NOT NULL,
+	latitude double(18,14) NOT NULL,
+	longitude double(18,14) NOT NULL,
+	rating float NOT NULL,
+	Monday char(11) NOT NULL,
+	Tuesday char(11) NOT NULL,
+	Wednesday char(11) NOT NULL,
+	Thursday char(11) NOT NULL,
+	Friday char(11) NOT NULL,
+	Saturday char(11) NOT NULL,
+	Sunday char(11) NOT NULL,
+	photourl text(512) NOT NULL,
+	take_out enum('YES', 'NO') NOT NULL,
```

Comment:
Consider use `TINYINT(1)` for boolean (`true`, `false`) data.

Reply:
We can modify all `enum('YES', 'NO')` below.

Reply:
@harrydrippin As I know, MySQL has BIT type. I think It's better for boolean data than TINIYINT.